ams/ams-api/src/main/java/com/netease/arctic/ams/api/client/OptimizeClient.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.ams.api.client;
+
+import com.netease.arctic.ams.api.OptimizeManager;
+import java.io.Serializable;
+
+public interface OptimizeClient extends OptimizeManager.Iface, Serializable {
```

Comment:
I don't see a reason to define this interface.
May you can use `OptimizeManagerClientPools` to connect to optimizing manager service.

Reply:
@hameizi cc

Reply:
Ou, I got the point, This interface defines some deviations。What I need is an independent interface named OptimizeClient, The implementation of OptimizeClient contain the "OptimizeManager.Iface" field, I don't want to use "OptimizeManager.Iface" directly